src/org/alicebot/ab/Bot.java

```diff
@@ -60,6 +60,35 @@
   private SraixHandler sraixHandler = null;
   public Locale locale = MagicBooleans.defaultLocale;
   
+  /**
+   * Set all directory path variables for this bot
+   *
+   * @param both_dir_path
+   *          root directory of the bot's folder
+   */
+  public void setAllPaths(String both_dir_path) {

```

Comment:
Lets rename this variable to something that makes sense..  like  "bot_root"  or something.  "both_dir_path" doesn't really mean anything ..